controllers/api/v1/token.go

```diff
@@ -36,6 +37,31 @@ func (c *Token) Create() {
 	c.serveJSON(&tokenResponse)
 }
 
+func (c *Token) Revoke() {
+	err := c.authenticateClient()
```

Comment:
I found in the [Doorkeeper] (https://github.com/doorkeeper-gem/doorkeeper/blob/main/app/controllers/doorkeeper/tokens_controller.rb#L5) that it validate client credentials also for revoking token. So I validated it here too.